.github/workflows/run-tests.yml

```diff
@@ -73,7 +72,7 @@ jobs:
       - name: Run Robot Tests
         run: |
           ${{ matrix.set_display }}
-          pabot --randomize suites -e BrokenORWindowsOREdge -v sauce_username:${{ secrets.SAUCE_USERNAME }} -v sauce_key:${{ secrets.SAUCE_ACCESS_KEY }} -v GITHUB_RUN_NUMBER:Github.Zoomba.$GITHUB_RUN_NUMBER test
```

Comment:
Duplicated here so that we rerun our tests in case of flakiness. Just gives us a 2nd chance to pass.

Reply:
What is duplicated exactly?